app/src/main/res/values-nn/strings.xml

```diff
@@ -0,0 +1,348 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
This is not needed

Reply:
It’s automatically generated by the tooling I use (android2po/`a2po`) to manage the translation. Is it a problem that it’s included? If so, I can of course manually remove it.

Reply:
Not really, it was just for parity with the other translations